docs/ci-cd-environment/aws-support.md

```diff
@@ -92,19 +105,23 @@ Note: if your key was encrypted using the default `aws/ssm` KMS key, `SEMAPHORE_
 
 ### 5. Bootstrap the CDK application
 
-The AWS CDK requires a few resources to be on hand for it to work properly:
+Using the ARN of the execution policy created in step 3, bootstrap the CDK application:
```

Comment:
Is there a command in `aws` cli to display this arn?

Reply:
Unfortunately, no.

There is a [`get-policy`](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/iam/get-policy.html), but that one takes an ARN 😢

Reply:
😢